_posts/2021-09-08-deep-dive-into-a-debezium-community-connector-scylla-cdc-source-connector.adoc

```diff
@@ -0,0 +1,229 @@
+---
+layout: post
+title: "Deep dive into a Debezium community connector: Scylla CDC Source Connector"
+date: 2021-09-08
+tags: [ community, kafka, scylla ]
+author: pgrabowski
+---
+
+At ScyllaDB, we develop a high-performance NoSQL database https://www.scylladb.com/[Scylla], API-compatible with Apache Cassandra, Amazon DynamoDB and Redis. Earlier this year, we introduced support for https://docs.scylladb.com/using-scylla/cdc/cdc-intro/[Change Data Capture] in Scylla 4.3. This new feature seemed like a perfect match for integration with the Kafka ecosystem, so we developed the https://github.com/scylladb/scylla-cdc-source-connector[Scylla CDC Source Connector] using the Debezium framework. In this blogpost we will cover the basic structure of Scylla’s CDC, reasons we chose the Debezium framework and design decisions we made.
```

Comment:
```suggestion
At ScyllaDB, we develop a high-performance NoSQL database https://www.scylladb.com/[Scylla], API-compatible with Apache Cassandra, Amazon DynamoDB and Redis. Earlier this year, we introduced support for https://docs.scylladb.com/using-scylla/cdc/cdc-intro/[Change Data Capture] in Scylla 4.3. This new feature seemed like a perfect match for integration with the Apache Kafka ecosystem, so we developed the https://github.com/scylladb/scylla-cdc-source-connector[Scylla CDC Source Connector] using the Debezium framework. In this blogpost we will cover the basic structure of Scylla’s CDC, reasons we chose the Debezium framework and design decisions we made.
```

The first reference should be qualified, adhering to the standard usage rules/recommendations.